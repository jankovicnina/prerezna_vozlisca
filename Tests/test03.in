20 37
1 5 -2.0
3 15 0.78
5 14 9.22
2 4 5.78
17 8 4.25
18 9 3.37
2 8 -1.4
4 12 0.14
17 18 -1.85
7 17 -0.32
16 4 -0.64
15 6 7.61
15 18 3.08
9 12 6.44
7 18 6.27
16 14 8.5
6 12 1.07
16 10 8.99
8 1 6.55
5 13 5.1
19 12 9.26
3 11 -0.04
9 7 0.83
13 3 5.13
10 11 0.62
14 16 5.59
4 14 6.37
11 6 0.88
2 1 7.17
15 19 4.32
3 9 -1.11
11 18 0.57
12 0 3.74
0 2 0.31
6 18 1.98
19 11 0.69
18 14 9.51
