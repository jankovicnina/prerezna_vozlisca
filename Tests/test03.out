11.49807894736842
